lib/main.dart

```diff
@@ -90,7 +100,10 @@ class MyHomePage extends StatelessWidget {
                         Expanded(
                           child: Center(
                             child: TextButton(
-                              onPressed: () => {},
+                              onPressed: () => {
+                                weather.value =
+                                    yumemiWeather.fetchSimpleWeather()
+                              },
```

Comment:
[nits]
このようにかけます 🙆‍♂️ 
`=>` をつけると若干意味が変わってくるので要注意です！
```suggestion
                              onPressed: () {
                                weather.value =
                                    yumemiWeather.fetchSimpleWeather();
                              },
```

Reply:
@blendthink 

失礼しました🙏（JSの書き方が抜けてないですね...）
4570af2 こちら修正しましたのでご確認よろしくお願いします！